Prepare for release  of sdk

Dr-Electron opened the pull request:
# Description of change

Once we are ready for the SDK release this PR should be merged.

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

lucas-tortora reviewed (approved):
LGTM, though we may want to merge https://github.com/iotaledger/iota-wiki/pull/1047 first. 

As well, we could consider keeping the old client docs and add an admonition recommending the sdk instead.

Dr-Electron commented:
> LGTM, though we may want to merge #1047 first.
> 
> As well, we could consider keeping the old client docs and add an admonition recommending the sdk instead.

Yes, i need to rebase 👍🏻

And yes we would do that. @luca-moser wdyt?

Dr-Electron commented:
/create-preview

Dr-Electron commented:
/create-preview

Dr-Electron commented:
Re-added the old docs and added deprecate admonitions:
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/iota.js/welcome/
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/iota.rs/welcome/
https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/wallet.rs/welcome/
Also added an announcement bar: https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/

lucas-tortora commented on shimmer/docusaurus.config.js:
```diff
@@ -15,6 +15,14 @@ module.exports = async () => {
+      announcementBar: {
+        id: 'iota-sdk',
+        content:
+          'The new IOTA SDK got released, <a target="_blank" href="/iota-sdk/welcome/">check it out</a>🚀. It deprecates iota.js, iota.rs and wallet.rs',
```
```suggestion
          'The new IOTA SDK got released, <a target="_blank" href="/shimmer/iota-sdk/welcome/">check it out</a>🚀. It deprecates iota.js, iota.rs, and wallet.rs',
```

or 

```suggestion
          '<a target="_blank" href="/shimmer/iota-sdk/welcome/">Check out the new IOTA SDK</a>. 🚀. It deprecates iota.js, iota.rs, and wallet.rs',
```

lucas-tortora commented on shimmer/docs/iota-sdk/docs/contribute.md:
```diff
@@ -0,0 +1,49 @@
+
+Helping others is a crucial aspect of any open source ecosystem. By sharing your knowledge with the community, you can
+provide immense value and potentially inspire others to learn and contribute. Join the ongoing discussions in the
+#clients-discussion channel on [Discord](https://discord.iota.org).
```
if we created a new channel we should probably update this

Dr-Electron replied:
```suggestion
#libraries channel on [Discord](https://discord.iota.org).
```

lucas-tortora commented:
We should probably update https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/develop/explanations/what-is-shimmer/clients/ as well. Do you want to do it in this PR or should I create an issue so we can do it in another?

Dr-Electron commented:
> We should probably update https://iota-wiki-poqn56oxe-iota1.vercel.app/shimmer/develop/explanations/what-is-shimmer/clients/ as well. Do you want to do it in this PR, or should I create an issue so we can do it in another?

As discussed I think it might make sense to create another PR for it

> We could also add the depreciation admonition here as a quick fix and create a new article. Though I think part of the benefit of the SDK is not having to explain which you should choose.

I think we can take the time and update the article. Should be doable today and not many people will this site as their first visit anyway :trollface:
